README.md

```diff
@@ -175,36 +179,16 @@ Creating and cloning a build:
       "container_name": "base",
       "config_file": "lxc.config",
       "init_timeout": "120s",
-      "template_parameters": [
-        "-d",
-        "ubuntu",
-        "-r",
-        "trusty",
-        "-a",
-        "amd64"
-      ],
-      "template_name": "ubuntu",
-      "cleanup_first": true
+      "rootfs": {
+        "archive": "/path/to/rootfs.tar.gz",
+        "config": "/path/to/lxc.config"
```

Comment:
I assume the only reason it doesn't is because there's a few system-specific things like the LXC path which make it not trivial to design around. A good idea to follow up though. Maybe we just toss the lxc.config in the archive outside of the plugin for a Sauce-specific usage?